.azure-pipelines/templates/checkout-code.yml

```diff
@@ -6,7 +6,7 @@ steps:
 # Azure checks out the code from the PR but leaves us in a detached state.
 # Because ddev cannot work in that state, we create a branch.
 # Only applies to PRs
-- script: git checkout -B $(System.PullRequest.SourceBranch)
+- script: git checkout -B azure/$(System.PullRequest.SourceBranch)
```

Comment:
About `git checkout -B`, seems not safe, if the branch already exist. 

So, probably better to do this instead:

```
git checkout -D azure/$(System.PullRequest.SourceBranch)
git checkout -b azure/$(System.PullRequest.SourceBranch)
```